lib/asciidoctor.rb

```diff
@@ -7,10 +7,9 @@
 
 require 'set'
 
-# NOTE RUBY_ENGINE == 'opal' conditional blocks are filtered by the Opal preprocessor
+# NOTE RUBY_ENGINE == 'opal' conditional blocks like this are filtered by the Opal preprocessor
 if RUBY_ENGINE == 'opal'
-  # NOTE asciidoctor/opal_ext is supplied by the Asciidoctor.js build
-  require 'asciidoctor/opal_ext'
+  require 'asciidoctor/opal' # supplies compile and runtime overrides for the Opal environment; fulfilled by Asciidoctor.js
```

Comment:
That's a good point. Then I think we'll need two includes. Perhaps at the beginning we have:

```
require 'asciidoctor/opal'
```

That's where you put all the stuff that is supplemental. Then, at the very end of the file we can have:

```
require 'asciidoctor/opal/coda'
```

That would be a "coda" where you can add overrides.

wdyt? Should we try it?